content/zh-cn/blog/_posts/2023-01-12-protect-mission-critical-pods-priorityclass/index.md

```diff
@@ -0,0 +1,524 @@
+---
+layout: blog
+title: "使用 PriorityClass 确保你的关键任务 Pod 免遭驱逐"
+date: 2023-01-12
+slug: protect-mission-critical-pods-priorityclass
+description: "Pod 优先级和抢占有助于通过决定调度和驱逐的顺序来确保关键任务 Pod 在资源紧缩的情况下正常运行。"
+---
+
+<!--
+layout: blog
+title: "Protect Your Mission-Critical Pods From Eviction With PriorityClass"
+date: 2023-01-12
+slug: protect-mission-critical-pods-priorityclass
+description: "Pod priority and preemption help to make sure that mission-critical pods are up in the event of a resource crunch by deciding order of scheduling and eviction."
+-->
+
+**作者**：Sunny Bhambhani (InfraCloud Technologies)
+
+<!--
+**Author:** Sunny Bhambhani (InfraCloud Technologies)
+-->
+
+**译者**：Wilson Wu (DaoCloud)
```

Comment:
AIUI, we usually don't credit localization teams in the page. We do appreciate the help though.

Reply:
OK, I saw most of other translation in the blogs have it, if need, I can remove it.